### ISSUE TITLE
If other instances of RBP_Support are active, it attempts to use their Templates instead

### ISSUE THREAD

**Issue by d4mation** [realbigplugins/rbp-support#5]
No freaking clue why this is happening. As far as I can tell, it should be using the directory of the plugin that is calling RBP_Support.

**Comment by d4mation** [comment]
Actually, no. This makes sense. Example:

- EDD iDevAffiliate has RBP_Support
- LearnDash Slack does too
- EDD iDevAffiliate loads the RBP_Support _FILE_ into memory first
- LearnDash Slack can override the template just fine, but if it attempts to load the default file it gets the default file from EDD iDevAffiliates directory since it is referencing `__DIR__`.

So this should ideally never be a problem, but it still should not do this. Going to need to find an elegant way around this. I'd rather not have to pass in the directory where you've included RBP_Support.

**Comment by d4mation** [comment]
Actually, the more I ponder this, this is a problem no matter what function it is. Even though the Prefix is utilized to target the correct Database Interactions based on the Plugin, the functions themselves that are doing those interactions would still be used from whatever Plugin utilizing RBP_Support gets loaded first. 

There isn't a way around this aside from potentially namespacing, but I don't think that can be done dynamically outside of something like Composer. I really like using this as a Git Submodule though and I'm not sure how Composer would react to pointing it at a Git Repo. It makes updating it as simple as `git pull` form the RBP_Support directory and committing the Commit Hash change to the Plugin's Repo.

So, in short, I think this a moot point. We will just need to ensure we update our Plugins Utilizing RBP_Support with the latest Tag for RBP_Support. Currently RBP_Support includes a list of all Plugins in the Support Form and it could be modified to include the version of the RBP_Support Object that sent it. That would tell help us know if some odd bug was happening due to one plugin expecting functionality available being in a certain version of RBP_Support not existing because it was loading an older version from another Plugin.

**Comment by d4mation** [comment]
> There isn't a way around this aside from potentially namespacing

Technically, a dynamic namespace per plugin could be created like this 😏 

https://github.com/realbigplugins/rbp-support/blob/fb2a7439608303a9bf0c7ce59d5977f98db5f135/rbp-support.php#L982-L991
